synapse/storage/background_updates.py

```diff
@@ -74,7 +73,7 @@ def total_items_per_ms(self):
             return float(self.total_item_count) / float(self.total_duration_ms)
 
 
-class BackgroundUpdateStore(SQLBaseStore):
+class BackgroundUpdater(object):
```

Comment:
given that a bunch of this guy's methods are now treated as public, they should probably lose their `_` prefixes. That can happen separately though.

Reply:
Yup, will do